Mariuzzo.DO.DataAnnotations/CedulaAttribute.cs

```diff
@@ -13,7 +13,10 @@ public sealed class CedulaAttribute : ValidationAttribute
     {
         
         private readonly IValidator _cedulaValidator = new CedulaValidator();
-
+        /// <summary>
+        /// Verify some exceptions of valid Cédula that don't follow the Luhn Algorithm
+        /// </summary>
+        public bool IgnoreExceptions = true;
```

Comment:
Ok, I was looking for a way to allow programmers to choose that option, but I think is better like you said, then the logic can be moved to the CedulaValidator besides doing it at the CedulaAttribute class. Let me refactor it. :)